MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -42435,34 +42510,65 @@
       "Id": "CollectibleStrayFairySnowhead1",
       "RequiredItems": [
         "AreaSnowheadTempleAccess",
-        "MaskGreatFairy",
+        "Enter Snowhead Temple",
         "Pass through false walls",
-        "Enter Snowhead Temple"
+        "SHT Iced Stairwell",
+        "MaskGreatFairy"
       ],
       "ConditionalItems": [
+        [
+          "Use Fire Arrow",
+          "MaskGoron"
+        ],
         [
           "1 Snowhead Key",
-          "SHT Zora Jumps",
-          "Use Fire Arrow"
+          "SHT Map Room Elevator"
```

Comment:
Is the jump to the locked door assumed to be done with Zora Mask without any tricks enabled? Or am I missing something for this one?

Reply:
I found myself writing an entry for that specific jump, but it ended up with so many conditionals that I decided to just put the jump into logic anyway since you can do it as human without too much effort. From now on, I would like SHT Zora Jumps to only refer to jumps Zora can do that regular Link cannot do.